06/Elena Davydik/src/components/Footer/About.tsx

```diff
@@ -0,0 +1,15 @@
+import { StyledList, StyledListItem } from "./index";
+
+export const About = () => {
+  return (
+    <StyledList>
+      <h5 className="list__title">About</h5>
+      <div className="active">
+        <StyledListItem>How Triphouse works</StyledListItem>
```

Comment:
we can use cycle here with array of strings

Reply:
I wrote config.ts with all text for footer and use arr.map for generate layout.